gen.go

```diff
@@ -0,0 +1,215 @@
+package avro
```

Comment:
I would suggest its own package like `gen`. Would be simple to then know what is happening with `gen.Struct()`?

Reply:
So rename `GenerateFrom` to `Struct`, did I get that right?

Call code then becomes `gen.Struct(...)`?

Reply:
Was an idea. I think the package `gen` makes the call code really clear: `gen.Xxxx()`. This leaves no guessing about  what  this would do. In this case as we define it must be a `Record` I would say `gen.Struct` or `gen.Record`. I am open to suggestion here.

Reply:
I like it. `Struct` is Go nomenclature and `Record` is avro nomenclature. Both seem fine to me and I'll throw a more verbose one `gen.StructFromRecord`, let me know which one you want, I'll change.

Reply:
Personally I would go for `Struct`. Short, sweet and is the output of the function.